soccer/planning/RotationCommand.hpp

```diff
@@ -23,16 +34,32 @@ struct FacePointCommand : public RotationCommand {
         : RotationCommand(FacePoint), targetPos(target) {}
 
     const Geometry2d::Point targetPos;
+
+    [[nodiscard]] std::string print() const override {
```

Comment:
Shouldn't this just be in the `operator<<`?

Reply:
What do you mean?

Reply:
NVM, github was showing diffs in the most inconvenient way possible. You're fine.